krod/core/llm_manager.py

```
@@ -69,13 +69,179 @@ def __init__(self, config: Dict[str, Any] = None):
+                "prompt_type": prompt_type
+            }
+            
+        except Exception as e:
+            return {
+                "success": False,
+                "error": str(e),
+                "prompt_type": prompt_type
+            }
+
+    def _call_openai_chat(
+        self, 
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)